parser/helper/helpers.go

```diff
@@ -172,3 +173,46 @@ func (h *Helper) GetMethodName(msg *parser.LotusMessage, height int64, key filTy
 func (h *Helper) GetEVMSelectorSig(ctx context.Context, selectorID string) (string, error) {
 	return h.actorCache.GetEVMSelectorSig(ctx, selectorID)
 }
+
+func (h *Helper) FilterTxsByActorType(ctx context.Context, txs []*types.Transaction, actorType string) ([]*types.Transaction, error) {
+	var result []*types.Transaction
+	for _, tx := range txs {
+		addrTo, err := address.NewFromString(tx.TxTo)
+		if err != nil {
+			h.logger.Sugar().Errorf("could not parse address. Err: %s", err)
+			return nil, err
+		}
+		addrFrom, err := address.NewFromString(tx.TxFrom)
+		if err != nil {
+			h.logger.Sugar().Errorf("could not parse address. Err: %s", err)
+			return nil, err
+		}
+
+		// TODO: TipsetKey
```

Comment:
Can we ask @raynaudoe what the implications about this are? I am not quite sure.